param_persist/serialize/serializer.py

```diff
@@ -57,6 +60,59 @@ def to_json(cls, param_object):
         param_json = json.dumps(param_dict)
         return param_json
 
+    @classmethod
+    def from_dict(cls, param_dict):
+        """
+        Create an instance of a param clam from a dictionary.
+
+        Args:
+            param_dict (dict): A dictionary representation of a param class.
+
+        Returns:
+            An instance of a param class defined by the dictionary.
+        """
+        class_path = param_dict.get('class_path')
+
+        if not class_path:
+            raise RuntimeError('Param not configured correctly. Missing "class_path" definition.')
+
+        dict_params = param_dict.get('params')
```

Comment:
Got yourself in a pickle with `param_dict` and `dict_params`, eh? We could use single-character arguments for the methods (e.g. `d` instead of `param_dict`).

Reply:
just renamed it params, which is what it is.